guacamole/src/main/webapp/app/groupList/types/GroupListItem.js

```diff
@@ -187,7 +187,7 @@ angular.module('groupList').factory('GroupListItem', ['ConnectionGroup', functio
             dataSource : dataSource,
 
             // Type information
-            expandable : includeSharingProfiles,
+            expandable : includeSharingProfiles !== false,
```

Comment:
Sharing profiles are supposed to be included by default (the function is documented as `includeSharingProfiles` defaulting to `true`), but not specifying that parameter makes `includeSharingProfiles` falsey. The `expandable` property needs to be `true` unless `includeSharingProfiles` is explicitly `false`.